test/analytics/analytics-on-redshift/lambda/custom-resources/create-schemas.test.ts

```diff
@@ -353,93 +214,55 @@ describe('Custom resource - Create schemas for applications in Redshift database
     expect(redshiftDataMock).toHaveReceivedCommandTimes(DescribeStatementCommand, 2);
   });
 
-  test('Created bi user with lower case characters only', async () => {
-    smMock.onAnyCommand().resolves({});
-    lambdaMock.on(ListTagsCommand).resolves({
-      Tags: { tag_key: 'tag_value' },
-    });
-    const regex = new RegExp(`^CREATE USER ${biUserNamePrefix}[a-z0-9A-Z$%]{8} PASSWORD '[a-zA-Z0-9!#$%^&-_=+|]{32}'$`);
-    redshiftDataMock.on(ExecuteStatementCommand).resolvesOnce({ Id: 'Id-1' })
-      .callsFakeOnce(input => {
-        if (input as ExecuteStatementCommandInput && regex.test(input.Sql)) {
-          return 'Id-2';
-        }
-        throw new Error(`Sql '${input.Sql}' are not expected, the bi user does not meet the pattern.`);
-      });
-    redshiftDataMock.on(BatchExecuteStatementCommand).resolves({ Id: 'Id-2' });
-    redshiftDataMock.on(DescribeStatementCommand).resolves({ Status: 'FINISHED' });
-    const resp = await handler(createServerlessEvent, context, callback) as CdkCustomResourceResponse;
-    expect(resp.Status).toEqual('SUCCESS');
-  });
-
   test('Created database, bi user, schemas and views in Redshift serverless', async () => {
     smMock.onAnyCommand().resolves({});
     lambdaMock.on(ListTagsCommand).resolves({
       Tags: { tag_key: 'tag_value' },
     });
     redshiftDataMock.on(ExecuteStatementCommand).resolves({ Id: 'Id-1' });
-    redshiftDataMock.on(BatchExecuteStatementCommand).callsFakeOnce(input => {
-      const sqlStr = input.Sqls.join(';\n');
-      if (input as BatchExecuteStatementCommandInput) {
-        if (sqlStr.includes('CREATE SCHEMA IF NOT EXISTS app1')) {
-          return { Id: 'Id-1' };
-        }
-      }
-      throw new Error('Sql-6 are not expected');
-    }).callsFakeOnce(input => {
-      if (input as BatchExecuteStatementCommandInput) {
-        for (const sql of input.Sqls) {
-          if (sql.includes('{{')) {
-            throw new Error(`The SQL '${sql}' contains the mustache variables!`);
-          }
-        }
-      }
-      return 'Id-22';
-    });
     redshiftDataMock.on(DescribeStatementCommand).resolves({ Status: 'FINISHED' });
+
     const resp = await handler(createServerlessEvent, context, callback) as CdkCustomResourceResponse;
+
     expect(resp.Status).toEqual('SUCCESS');
-    expect(redshiftDataMock).toHaveReceivedCommandTimes(ExecuteStatementCommand, 2);
-    expect(redshiftDataMock).toHaveReceivedCommandTimes(BatchExecuteStatementCommand, 2);
-    expect(redshiftDataMock).toHaveReceivedNthSpecificCommandWith(1, BatchExecuteStatementCommand, {
-      Sqls: expect.arrayContaining([
-        expect.stringMatching(`GRANT USAGE ON SCHEMA app1 TO ${biUserNamePrefix}\\w{8}`),
-      ]),
-    });
+    expect(redshiftDataMock).toHaveReceivedCommandTimes(ExecuteStatementCommand, 44);
```

Comment:
Let's have a better way to get this number, for example, viewCount * 2 + reportingCount * 2 + biUserSQLCount + databaseSQLCount

Reply:
chagned